fortios/provider.go

```diff
@@ -45,30 +45,31 @@ func Provider() terraform.ResourceProvider {
 		},
 
 		ResourcesMap: map[string]*schema.Resource{
-			"fortios_networking_route_static":      resourceNetworkingRouteStatic(),
-			"fortios_networking_interface_port":    resourceNetworkingInterfacePort(),
-			"fortios_system_admin_profiles":        resourceSystemAdminProfiles(),
-			"fortios_system_admin_administrator":   resourceSystemAdminAdministrator(),
-			"fortios_firewall_object_address":      resourceFirewallObjectAddress(),
-			"fortios_firewall_object_addressgroup": resourceFirewallObjectAddressGroup(),
-			"fortios_firewall_object_service":      resourceFirewallObjectService(),
-			"fortios_firewall_object_servicegroup": resourceFirewallObjectServiceGroup(),
-			"fortios_firewall_object_vip":          resourceFirewallObjectVip(),
-			"fortios_firewall_object_vipgroup":     resourceFirewallObjectVipGroup(),
-			"fortios_firewall_object_ippool":       resourceFirewallObjectIPPool(),
-			"fortios_firewall_security_policy":     resourceFirewallSecurityPolicy(),
-			"fortios_system_setting_global":        resourceSystemSettingGlobal(),
-			"fortios_system_setting_dns":           resourceSystemSettingDNS(),
-			"fortios_system_setting_ntp":           resourceSystemSettingNTP(),
-			"fortios_log_syslog_setting":           resourceLogSyslogSetting(),
-			"fortios_log_fortianalyzer_setting":    resourceLogFortiAnalyzerSetting(),
-			"fortios_system_apiuser_setting":       resourceSystemAPIUserSetting(),
-			"fortios_system_vdom_setting":          resourceSystemVdomSetting(),
-			"fortios_system_license_forticare":     resourceSystemLicenseFortiCare(),
-			"fortios_system_license_vdom":          resourceSystemLicenseVDOM(),
-			"fortios_system_license_vm":            resourceSystemLicenseVM(),
-			"fortios_vpn_ipsec_phase1interface":    resourceVPNIPsecPhase1Interface(),
-			"fortios_vpn_ipsec_phase2interface":    resourceVPNIPsecPhase2Interface(),
+			"fortios_networking_route_static":          resourceNetworkingRouteStatic(),
+			"fortios_networking_interface_port":        resourceNetworkingInterfacePort(),
+			"fortios_system_admin_profiles":            resourceSystemAdminProfiles(),
+			"fortios_system_admin_administrator":       resourceSystemAdminAdministrator(),
+			"fortios_firewall_object_address":          resourceFirewallObjectAddress(),
+			"fortios_firewall_object_addressgroup":     resourceFirewallObjectAddressGroup(),
+			"fortios_firewall_object_service":          resourceFirewallObjectService(),
+			"fortios_firewall_object_service_category": resourceFirewallObjectServiceCategory(),
```

Comment:
Please use 'fortios_firewall_object_servicecategory' instead of 'fortios_firewall_object_service_category' here, we are trying to limit the name len no bigger then 4 paragraph.